cookbooks/aws-parallelcluster-platform/kitchen.platform-config.yml

```diff
@@ -48,22 +48,27 @@ suites:
             vpc_ipv4_cidr_blocks: |
               cidr1
               cidr2
-  - name: fabric_manager
+  - name: nvidia_fabric_manager
     run_list:
       - recipe[aws-parallelcluster-tests::setup]
       - recipe[aws-parallelcluster-tests::test_resource]
     verifier:
       controls:
         - /tag:config_.*fabric_manager/
     attributes:
+      dependencies:
+        - resource:package_repos
+        - resource:install_packages:install_kernel_source
+        - 'resource:package { "package_name": "dkms" }'
+        - resource:build_tools
+        - recipe:aws-parallelcluster-platform::nvidia_install
       resource: fabric_manager:configure
       cluster:
         nvidia:
           enabled: true
-      dependencies:
-        - resource:fabric_manager:setup
```

Comment:
are we sure we don't need the resource setup phase to test the fabric_manager:configure step?

Reply:
That action is executed in the nvidia_install dep